NamingSystems/ods-organization-code-namingsystem.xml

```diff
@@ -0,0 +1,36 @@
+<NamingSystem xmlns="http://hl7.org/fhir">
+  <id value="ods-organization-code-namingsystem" />
+  <name value="ODSOrganisationCode" />
```

Comment:
Name should be in the format of England[Name]

Reply:
Namingsystem removed